lib/mix/tasks/nettools.ex

```diff
@@ -41,25 +43,38 @@ defmodule Mix.Tasks.Archethic.Nettools do
     |> args_to_internal_representation()
   end
 
+  # For switch --help or -h
   def args_to_internal_representation({[help: true], [], []}) do
     Mix.shell().cmd("mix help #{Mix.Task.task_name(__MODULE__)}")
   end
 
-  def args_to_internal_representation({[punch: true], _port_list = [port | more_ports], errors}) do
-    Networking.PortForwarding.try_open_port(abs(Integer.parse(port) |> elem(0)), true)
-    args_to_internal_representation({[punch: true], more_ports, errors})
+  # For switch --punch or -p Opens default http and p2p ports
+  def args_to_internal_representation({[punch: true], _port_list = [], _errors}) do
```

Comment:
For the punch you also need to set the env, as it's checking IP lookup or enforced by conf